src/bin/inx-chronicle/cli.rs

```diff
@@ -254,4 +272,16 @@ pub enum Subcommands {
         #[arg(short, long)]
         num_tasks: Option<usize>,
     },
+    #[cfg(debug_assertions)]
+    ClearDatabase {
+        #[arg(short, long)]
+        no_run: bool,
+    },
+    BuildIndexes,
+}
+
+#[derive(Copy, Clone, PartialEq, Eq)]
+pub enum PostCommand {
+    Start,
+    Exit,
 }
```

Comment:
[nitpick] Should we use [`std::ops::ControlFlow`](https://doc.rust-lang.org/std/ops/enum.ControlFlow.html) here?

Reply:
Eh? I don't really care, but that enum has more than we need for this.

Reply:
At some point we might want to transport information to the main process though. We can just set `B` and `C` to `()` for now.